### PR TITLE
Fix: add icons to features section

### PR DESCRIPTION
**Pull request by dev-phantom**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] ⏩ Revert

## Description

Added icons by creating two new folder and a single file
also  changed the color of the h3

## Related Tickets & Documents

Fixes: #22 
## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### 📸 Credits

This pull request template was created by [Open Sauced](https://github.com/open-sauced) 🍕

### CODE REVIEW

**Comment by TAKANOME-DEV** [comment]
Hey @dev-phantom, is there a reason why you didn't use react icons?

**Comment by dev-phantom** [comment]
Yes 
Didn't know the names of the icons

File: apps/client/src/pages/index.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import Image from 'next/image';
-import { Button } from 'atoms';
+// import { Button } from 'atoms';
+
+import {
+    feedback,
+    collaboration,
+    learning,
+    community,
+} from "../assets/index"
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:9]:
```suggestion
} from "@/assets/index"
```

File: apps/client/src/pages/index.tsx [hunk 2]
```diff
@@ -54,11 +61,13 @@ export default function IndexPage() {
               Key features
             </span>
           </div>
-          <div className="grid grid-cols-2 gap-y-20 rounded-lg bg-slate-100 px-8 py-16 shadow">
-            <div className="flex max-w-lg gap-8 border border-red-500">
-              <div>Icon</div>
+          <div className="grid grid-cols-2 gap-y-20 rounded-lg bg-card px-20 py-16 shadow">
+            <div className="flex max-w-lg gap-8">
+              <div>
+                <Image src={feedback} alt="feeback" className='w-[40px]'/>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:67]:
```suggestion
                <Image src={feedback} alt="feedback and critique" width={120} height={120} />
```

File: apps/client/src/pages/index.tsx [hunk 3]
```diff
@@ -54,11 +61,13 @@ export default function IndexPage() {
...
+              </div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:70]:
```suggestion
                <h3 className="mb-4 text-slate-900 text-xl font-bold capitalize">Feedback and critique</h3>
```

File: apps/client/src/pages/index.tsx [hunk 4]
```diff
@@ -67,9 +76,11 @@ export default function IndexPage() {
               </div>
             </div>
             <div className="flex  max-w-lg  gap-8">
-              <div>Icon</div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <Image src={collaboration} alt="feeback" className='w-[40px]'/>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:80]:
```suggestion
                <Image src={collaboration} alt="collaboration and support" width={120} height={120} />
```

File: apps/client/src/pages/index.tsx [hunk 5]
```diff
@@ -67,9 +76,11 @@ export default function IndexPage() {
...
+              </div>
+              <div>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:83]:
```suggestion
                <h3 className="mb-4 text-slate-900 text-xl font-bold capitalize">Collaboration and support</h3>
```

File: apps/client/src/pages/index.tsx [hunk 6]
```diff
@@ -67,9 +76,11 @@ export default function IndexPage() {
...
                 <p>
                   Get constructive feedback and critiques from other writers and
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:85]:
```suggestion
                  Connect with other writers and bloggers to share ideas, offer support, and collaborate on projects.
```

File: apps/client/src/pages/index.tsx [hunk 7]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
               </div>
             </div>
             <div className="flex  max-w-lg  gap-8">
-              <div>Icon</div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <Image src={learning} alt="feeback" className='w-[40px]'/>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:93]:
```suggestion
                <Image src={learning} alt="learning resources" width={120} height={120} />
```

File: apps/client/src/pages/index.tsx [hunk 8]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
...
+              </div>
+              <div>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:96]:
```suggestion
                <h3 className="mb-4 text-slate-900 text-xl font-bold capitalize">Learning resources</h3>
```

File: apps/client/src/pages/index.tsx [hunk 9]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
...
                 <p>
                   Get constructive feedback and critiques from other writers and
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:98]:
```suggestion
                  Access a wide range of resources and guides to help you learn more about blogging, writing, and online publishing.
```

File: apps/client/src/pages/index.tsx [hunk 10]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
...
                   experts in your niche to help you improve your skills and grow
                   as a blogger.
                 </p>
               </div>
             </div>
-            <div className="flex  max-w-lg  gap-8">
-              <div>Icon</div>
+            <div className="flex max-w-lg  gap-8">
+              <div>
+                <Image src={community} alt="feeback" className='w-[40px]'/>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:106]:
```suggestion
                <Image src={community} alt="community forums" width={120} height={120} />
```

File: apps/client/src/pages/index.tsx [hunk 11]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
...
+              </div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:109]:
```suggestion
                <h3 className="mb-4 text-slate-900 text-xl font-bold capitalize">Community forums</h3>
```

File: apps/client/src/pages/index.tsx [hunk 12]
```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
...
                 <p>
                   Get constructive feedback and critiques from other writers and
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:111]:
```suggestion
                  Participate in forums and discussion groups to connect with other writers and bloggers, share your work, and get feedback.
```

**Review by TAKANOME-DEV — CHANGES_REQUESTED** [review]
Hey @dev-phantom just some minors changes and it's ready to go :rocket:

File: apps/client/src/pages/index.tsx (hunk 12 above)
Comment (dev-phantom) [reply apps/client/src/pages/index.tsx:111]: Okay noted 

The keyboard of my PC is currently damaged I'll resolve it when i fix it.

**Comment by dev-phantom** [comment]
these issue has been resolved in this pull request #34

**Pull request closed without merging**